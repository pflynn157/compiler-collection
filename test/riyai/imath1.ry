func main -> i32 is
    print(10 + 2);
    print(10 - 2);
    print(10 * 2);
    print(10 / 2);
    print(10 % 2);
    print(10 % 3);
    print(5 & 4);
    print(5 | 4);
    print(5 ^ 4);
    return 0;
end

