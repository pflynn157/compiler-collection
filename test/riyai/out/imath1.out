12
8
20
5
0
1
4
5
1
